src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1397,7 +1400,8 @@ static Library()
                 StandardErrorHandling<FormulaValue>(
                     BuiltinFunctionsCore.TimeZoneOffset.Name,
                     expandArguments: NoArgExpansion,
-                    replaceBlankValues: DoNotReplaceBlank,
+                    replaceBlankValues: ReplaceBlankWith(
+                        new DateTimeValue(IRContext.NotInSource(FormulaType.DateTime), _epoch)),
```

Comment:
Does this need to be update on Power Apps side as well? So I am assuming that this will be under excel compatibility change.

Reply:
Yes. Some of those issues are already fixed in Power Apps, others have issues. We'll update them when we are bringing the latest Power Fx bits there.